src/time.js

```diff
@@ -9,6 +9,20 @@ function advanceBlock () {
   });
 }
 
+// Advance the block to the passed height
+async function advanceBlockTo (target) {
+  if (!BN.isBN(target)) {
+    target = new BN(target);
+  }
+
+  let now = (await latestBlock());
+  if (target.lt(now)) throw Error(`Cannot advance current block (${now}) to a moment in the past (${target})`);
+  for (;target.gte(now);) {
+    await advanceBlock();
+    now = (await latestBlock());
```

Comment:
We can avoid a round-trip here
```suggestion
    now += 1;
```

Reply:
I don't think we should make this change this. We can't be sure that this process is the only one interacting with the network.

Reply:
Yes, I don't think so.